src/components/PostProcessedApiSpec/PostProcessedApiSpec.tsx

```diff
@@ -39,13 +32,17 @@ const PostProcessedApiSpec: React.FC = () => {
       ) : error ? (
         <S.ErrorLabel>{error.message}</S.ErrorLabel>
       ) : (
-        data && <SwaggerUI spec={parsedSpec} plugins={[TableOfContentsPlugin, DynamicServersPlugin]} />
+        data && <SwaggerUI spec={parseSpec(rawApiSpec)} plugins={[TableOfContentsPlugin, DynamicServersPlugin]} />
       )}
     </S.PostProcessedApiSpecContainer>
   );
 };
 
 const parseSpec = (spec: any) => {
```

Comment:
Would be better not to call this function knowing that the spec can be anything, better to check the value before calling this function